docs/curation-guidelines.md

```diff
@@ -12,9 +12,9 @@ This document sets out the minimum processes the ClearlyDefined community follow
 There are five possible license property values on a definition element:
 
 1. A valid SPDX license expression.
-2. The string `NOSASSERTION`. This indicates that license-like data is found, but that ClearlyDefined cannot identify a SPDX-identified license.
+2. The string `NOASSERTION`. This indicates that license-like data is found, but that ClearlyDefined cannot identify a SPDX-identified license.
 3. The string `NONE`. This indicates that a human confirmed that there is no license information in the file.
-4. The string `OTHER`. This indicates that a human confirmed that there is license information in the file but that the license is not an SPDX-identified license.)
+4. The string `OTHER`. This indicates that a human confirmed that there is license information in the file but that the license is not a SPDX-identified license.
```

Comment:
The change in this line is wrong. The usage of "an" depends on how something is spoken, not written: "an ess-pi-di-ex..."

Reply:
I have heard it spoken as an acronym, "spidex", which is why I thought bullet #2 was correct with "a". I'm guessing that isn't the common way it is spoken, so I will update my PR.